.gitlab-ci.yml

```diff
@@ -976,7 +976,10 @@ cudamemcheck:
   stage: QoS_tools
   extends:
     - .deploy_condition
-    - .use_gko-cuda110-gnu9-llvm9-intel2020
+  image: ginkgohub/cuda:101-gnu8-llvm10-intel2019
+  tags:
+    - private_ci
+    - nvidia-gpu
```

Comment:
Could you move it to image.yml?

Reply:
No, because I don't want to move the other jobs of that image to amdci. This is only temporary until we have more GPU-capable CI resources.

Reply:
if we are only using it for this, I am fine with it